content/patterns/federated-edge-observability/getting-started.adoc

```diff
@@ -0,0 +1,304 @@
+---
+title: Getting Started
+weight: 10
+aliases: /federated-edge-observability/getting-started/
+---
+
+:toc:
+:imagesdir: /images
+:_content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
+[id="deploying-fe-pattern"]
+== Deploying the {fe-pattern}
+
+.Prerequisites
+
+* An OpenShift cluster
+ ** To create an OpenShift cluster, go to the https://console.redhat.com/[Red Hat Hybrid Cloud console].
+ ** Select *OpenShift \-> Red Hat OpenShift Container Platform \-> Create cluster*.
+* A GitHub account and a token for it with repositories permissions, to read from and write to your forks.
+* The Helm binary, see link:https://helm.sh/docs/intro/install/[Installing Helm]
+For installation tooling dependencies, see link:https://validatedpatterns.io/learn/quickstart/[Patterns quick start].
+
+[id="preparing-for-deployment"]
+== Preparing for deployment
+.Procedure
+
+. Fork the link:https://github.com/validatedpatterns-sandbox/federated-edge-observability[federated-edge-observability] repository on GitHub. You must fork the repository because your fork is updated as part of the GitOps and DevOps processes.
+
+. Clone the forked copy of this repository.
++
+[source,terminal]
+----
+$ git clone git@github.com:<your-username>/federated-edge-observability.git
+----
+
+. Go to your repository: Ensure you are in the root directory of your Git repository by using:
++
+[source,terminal]
+----
+$ cd /path/to/your/repository
+----
+
+. Run the following command to set the upstream repository:
++
+[source,terminal]
+----
+$ git remote add -f upstream git@github.com:validatedpatterns-sandbox/federated-edge-observability.git
+----
+
+. Verify the setup of your remote repositories by running the following command:
++
+[source,terminal]
+----
+$ git remote -v
+----
++
+.Example output
++
+[source,terminal]
+----
+origin	git@github.com:kquinn1204/federated-edge-observability.git (fetch)
+origin	git@github.com:kquinn1204/federated-edge-observability.git (push)
+upstream	git@github.com:validatedpatterns-sandbox/federated-edge-observability.git (fetch)
+upstream	git@github.com:validatedpatterns-sandbox/federated-edge-observability.git (push)
+----
+
+. Create a local copy of the Helm values file that can safely include credentials.
```

Comment:
We have separated the Helm values from the secrets. It would be more accurate to say something like this:

"Make a local copy of secrets template outside of your repository to hold credentials for the pattern."